lib/Dialect/TTNN/Transforms/TTNNWorkarounds.cpp

```diff
@@ -35,15 +35,9 @@ namespace mlir::tt::ttnn {
 #define GEN_PASS_DEF_TTNNWORKAROUNDS
 #include "ttmlir/Dialect/TTNN/Transforms/Passes.h.inc"
 
-// Helper method to get the tensor layout attribute from the op operand.
-static TTNNLayoutAttr getLayoutAttrFromOpOperand(OpOperand &opOperand) {
-  auto tensorType = mlir::cast<RankedTensorType>(opOperand.get().getType());
-  return mlir::cast<TTNNLayoutAttr>(tensorType.getEncoding());
-}
-
-// Helper method to get the tensor layout attribute from the op result.
-static TTNNLayoutAttr getLayoutAttrFromOpResult(OpResult &opResult) {
-  auto tensorType = mlir::cast<RankedTensorType>(opResult.getType());
+// Helper method to get the tensor layout attribute from the value.
```

Comment:
At some point we should move this to some utils header since it's used throughout the code.